automation/taskcluster/lib/tasks.py

```diff
@@ -427,7 +439,7 @@ def craft_upload_apk_nimbledroid_task(self, assemble_task_id):
             name="Upload Release APK to Nimbledroid",
             description='Upload APKs to Nimbledroid for performance measurement and tracking.',
             command=' && '.join([
-                'curl --location https://queue.taskcluster.net/v1/task/{}/artifacts/public/target.arm.apk > target.arm.apk'.format(assemble_task_id),
+                'curl --location "{}/{}/artifacts/public/target.arm.apk" > target.arm.apk'.format(_DEFAULT_TASK_URL, assemble_task_id),
```

Comment:
Should we query using the `taskcluster` library to get this URL?
I suppose you're just modifying work from previous PRs, so :man_shrugging: :+1:

Reply:
As far as I know, the `taskcluster` lib doesn't expose the final URL, which we want here. I might be wrong, though.